app/global.css

```diff
@@ -36,6 +36,7 @@ input[type="checkbox"] {
     font-weight: 700;
     margin-bottom: 10px;
     text-align: center;
+    margin-top: 48px;
```

Comment:
Please align without hardcoding the margins :)

Reply:
I've updated the implementation to align the form to center without hardcoding margins. Please review and let me know if further adjustments are needed.